__tests__/app.test.js

```diff
@@ -31,3 +31,30 @@ describe("/api/categories", () => {
     return request(app).get("/api/categorieeeees").expect(404);
   });
 });
+
+describe("/api/reviews", () => {
+  test.only("GET: should respond with a 200 status code, and correctly formatted data", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews.length).toBe(reviewData.length);
+        const reviewOutput = {
+          owner: expect.any(String),
+          title: expect.any(String),
+          review_id: expect.any(Number),
+          category: expect.any(String),
+          review_img_url: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          designer: expect.any(String),
+          comment_count: expect.any(String),
+        };
+        reviews.forEach((review) => {
+          expect(review).toMatchObject(reviewOutput);
+        });
```

Comment:
Could we just add something in this test to check articles are sorted in date descending order?

Reply:
Yep, have done now - should I submit a new nchelp pr tomorrow?